### DISCUSSION TITLE
HyperHDR + Vero4k+

### DISCUSSION THREAD

**Discussion by NjayNL** [awawa-dev/HyperHDR#837] [General]
Hi, 

I am currently running hyperion 2.0.15 using the AMlogic local capture, however i am facing issues with my hue lights not display the correct color. is there a way to install HyperHDR on my vero4k+? Can i use the AMlogic local grabber for this?

**Comment by awawa-dev** [comment]
Hi
If your vero4k+ is using OS based on Debian bullseye or bookworm then you should be able to install HyperHDR using released DEB packages. Rather USB grabber is necessary since HyperHDR doesn't have AMlogic grabber (not sure if FrameGrabber will work on this platform).

**Reply by NjayNL** [reply]
Well i can run the hyperion installer from their site if i choose linux with the command sudo apt-get install hyperion 
So i am guessing i can install hyperhd too. Thanks for your reply :)

**Reply by awawa-dev** [reply]
If it's older Debian Buster then it wont work. Anyway HyperHDR installer should check the glibc version.

**Reply by NjayNL** [reply]
Thanks, the reason for wanting to switch to hyperhdr is because hyperion.ng is sending the wrong colors to my hue lights.

**Reply by NjayNL** [reply]
i tried installing and it works, but when it starts i get: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory 

my linux skills are so so. can you help?

it runs: 4.9.269-44-osmc

**Reply by awawa-dev** [reply]
which version did you install? execute `lsb_release -a` , `uname -m` , `file /usr/bin/hyperhdr`

**Reply by NjayNL** [reply]
that command does not work but the installer says 

Note, selecting 'hyperhdr:arm64' instead of './HyperHDR-20.0.0.0-Linux-aarch64.deb'

**Reply by awawa-dev** [reply]
none of these 3 commands work???

**Reply by NjayNL** [reply]
uname - m : gives aarch64

and

Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye

**Reply by awawa-dev** [reply]
so https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-aarch64.deb is probably right choice. I dont know why it cant find libstdc++.so.6
execute `file /usr/bin/hyperhdr` and `ldd /usr/bin/hyperhdr`

**Reply by NjayNL** [reply]
yes but the installer does this:

Note, selecting 'hyperhdr:arm64' instead of './HyperHDR-20.0.0.0-Linux-aarch64.deb'

**Reply by NjayNL** [reply]
GOT IT 

wget https://github.com/awawa-dev/HyperHDR/releases/download/v20.0.0.0/HyperHDR-20.0.0.0-Linux-armv6l.deb

sudo apt install -f ./HyperHDR-20.0.0.0-Linux-armv6l.deb

and it works :) 

thanks so much for your help!!

**Reply by awawa-dev** [reply]
> uname - m : gives aarch64

it's clearly aarch64 OS not 32 bit arm6l OS. Anyway it's good that it works finally.
